app/controllers/cohorts/notes_controller.rb

```diff
@@ -31,7 +31,7 @@ def create
       @note.cohort_client.touch
       # send notifications
       if note_params[:send_notification].present? && note_params[:recipients].present?
-        token = Token.tokenize(cohort_cohort_client_client_notes_path(@cohort, @cohort_client))
+        token = Token.tokenize(cohort_cohort_client_cohort_client_notes_path(@cohort, @cohort_client))
```

Comment:
This note is restricted to a specific cohort, so we send the user to the non-modal view of the list of cohort notes for that client.

Reply:
Correct.